src/Microsoft.ML.Vision/ImageClassificationTrainer.cs

```diff
@@ -389,7 +389,7 @@ public sealed class Options : TrainerInputBaseWithLabel
             /// Callback to report statistics on accuracy/cross entropy during training phase.
             /// </summary>
             [Argument(ArgumentType.AtMostOnce, HelpText = "Callback to report metrics during training and validation phase.", SortOrder = 15)]
-            public Action<ImageClassificationMetrics> MetricsCallback = null;
+            public Action<ImageClassificationMetrics> MetricsCallback = (metrics) => Console.WriteLine(metrics);
```

Comment:
The ML.NET library shouldn't be writing to the console. We are going to be used in a lot more places than just a console app.

Reply:
how about to stdout all the time?

Reply:
I don't view this as any different than the `MLContext.Log` property. We don't dump information to the console in it by default. I don't think we should be here either.

Reply:
Addressed this in my recent commit.